go.mod

```diff
@@ -1,11 +1,22 @@
 module github.com/stripe/skycfg
 
+go 1.15
+
 require (
+	github.com/cncf/udpa/go v0.0.0-20200909154343-1f710aca26a9 // indirect
+	github.com/envoyproxy/go-control-plane v0.9.6 // indirect
```

Comment:
It looks like something in your dev environment got confused about which `go.mod` to update. It also touched a bunch of files in the other examples. Can you revert these unrelated changes?